tools/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.core</groupId>
+    <artifactId>org.openhab.core.reactor</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.openhab</groupId>
```

Comment:
Hm, most stuff in this repo (as soon as the tycho leftovers used by addons can be removed) if using the group ID prefix `org.openhab.core`.
Why can't this be used for the archetype?

IMHO: The group ID is an ID "who" provides something, not "who" consumes something.
So, as long as this "tooling" is part of openHAB Core it should be use that group ID.
If it is not part of core at all, why placing it in this repo?

Also for the other fields (e.g. name, artifact ID, ...) it would be nice to use the format used in the other POM files (e.g. `openHAB Core :: ...`).

Reply:
I didn't pay a lot of attention to the naming, so copied basically what was in there. I've updated the groupIds, artifactIds and names. But you might want to have a second look to see if that was correctly done. Otherwise please do suggest how you think it should be named.